### PR TITLE
TELCODOCS-1619-414 Document Telco 4.14 late known issues RNs

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1619]: Document Telco 4.14 late known issues RNs

Version(s): 4.14 Release notes

Issue:https://issues.redhat.com/browse/TELCODOCS-1619

Link to docs preview:https://67571--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -2986,6 +2986,46 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
 
 * For {sno-caps} on {product-version} and Google Cloud Platform (GCP), there is a known issue with the Cloud Network Config Controller (CNCC) entering a `CrashLoopBackOff` state. This occurs at initialization time when the CNCC tries to reach the GCP internal load balancer address and the resulting hairpin traffic is not correctly prevented in OVN-Kubernetes shared gateway mode on GCP causing it to get dropped. Cluster Network Operator will show a `Progressing=true` status in such case. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
 
+* Large latency spikes may be seen whenever a container is started, on an {sno-caps} with guaranteed cpus and load balancing disabled. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
```
Comment (djuran) [inline release_notes/ocp-4-14-release-notes.adoc:2989]: Would be good to specify what load balancing we refer to.
I.e. what is it that is load balanced or not

Comment (kquinn1204) [reply release_notes/ocp-4-14-release-notes.adoc:2989]: Good point it is irq-load-balancing.crio.io to be honest there is an open question on whether I can report as noted as Red Hat internal

Comment (kquinn1204) [reply release_notes/ocp-4-14-release-notes.adoc:2989]: @djuran changed to _with guaranteed CPUs and Interrupt Request (IRQ) load balancing is disabled_

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -2986,6 +2986,46 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
 
 * For {sno-caps} on {product-version} and Google Cloud Platform (GCP), there is a known issue with the Cloud Network Config Controller (CNCC) entering a `CrashLoopBackOff` state. This occurs at initialization time when the CNCC tries to reach the GCP internal load balancer address and the resulting hairpin traffic is not correctly prevented in OVN-Kubernetes shared gateway mode on GCP causing it to get dropped. Cluster Network Operator will show a `Progressing=true` status in such case. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
 
+* Large latency spikes may be seen whenever a container is started, on an {sno-caps} with guaranteed CPUs and Interrupt Request (IRQ) load balancing is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-14-release-notes.adoc:2989]:
```suggestion
* On an {sno-caps} that has guaranteed CPUs and where Interrupt Request (IRQ) load balancing is disabled, large latency spikes can occur at container startup. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
```

Just a thought--attempting to make it easier to catch a note applicable to one's situation while scanning.

Not sure if I restructured the restrictive clause correctly.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -2986,6 +2986,46 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
...
+
+* When deploying an application with a large number of pods, some of which have CPU limits configured, the deployment can sometimes fail. The workaround is to re-deploy the application. (link:https://issues.redhat.com/browse/RHEL-7232[*RHEL-7232*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-14-release-notes.adoc:2991]:
```suggestion
* When deploying an application that has a large number of pods, some of which have CPU limits configured, the deployment can fail. The workaround is to re-deploy the application. (link:https://issues.redhat.com/browse/RHEL-7232[*RHEL-7232*])
```

Comment (maxwelldb) [reply release_notes/ocp-4-14-release-notes.adoc:2991]: ISG asks for "that" w/restrictive clauses.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -2986,6 +2986,46 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
...
+
+* On an {sno-caps} with disabled capabilities, the `openshift-controller-manager-operator` may continuously get restarted.  The workaround is to enable the build capability or to manually create the `builds.config.openshift.io` CRD.
```
Comment (maxwelldb) [inline release_notes/ocp-4-14-release-notes.adoc:2993]:
```suggestion
* On an {sno-caps} that has disabled capabilities, the `openshift-controller-manager-operator` may continuously restart.  As a workaround, enable the build capability or  manually create the `builds.config.openshift.io` CRD.
```

Comment (maxwelldb) [reply release_notes/ocp-4-14-release-notes.adoc:2993]: Will "enable the build capability" make sense to the audience?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 5]
```diff
@@ -2986,6 +2986,46 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
...
++
+Perform the following steps to manually create the `builds.config.openshift.io` CRD:
++
+--
+. Extract the release manifests:
```
Comment (maxwelldb) [inline release_notes/ocp-4-14-release-notes.adoc:2998]: Recall the "run the following command" language required by the repo contribution guidelines.

**Review by maxwelldb — COMMENTED** [review]
Left some comments and questions. Relabel when ready.

**Pull request merged**